### DISCUSSION TITLE
nextflow run hangs at wget

### DISCUSSION THREAD

**Discussion by brucemoran** [nextflow-io/nextflow#5107] [Q&A] [answered]
Hi,

I am running Nextflow in an offline Ubuntu VM.

I downloaded `nextflow-24.04.2-all` release, run `bash nextflow-24.04.2-all` which runs and gives help as expected.

I have `mv nextflow-24.04.2-all /usr/local/bin/nextflow`, and it is executable.

I tested `NXF_DEBUG=x nextflow run hello` and it hangs at:
```
++wget 'https://www.nextflow.io/releases/latest/version?current=24.04.2'
```
presumably because I do not have internet access.

Can I turn off this attempted `wget`?

I thought the `all` release was for weird offline use cases like mine.

Help much appreciated, if you want more info let me know.

Thanks, 

Bruce.

```
nextflow info

  Version: 24.04.2 build 5914
  Created: 29-05-2024 06:19 UTC (07:19 IST)
  System: Linux 6.2.0-39-generic
  Runtime: Groovy 4.0.21 on Java HotSpot(TM) 64-Bit Server VM 17.0.10+11-LTS-240
  Encoding: UTF-8 (UTF-8)
```

**Comment by pditommaso** [comment, marked as answer]
Yes, use the environment variable `NXF_OFFLINE=true`
